dev-libs/libcdio/libcdio-2.0.0.recipe

```diff
@@ -0,0 +1,107 @@
+SUMMARY="A library to encapsulate CD-ROM reading and control"
+DESCRIPTION="The libcdio package contains a library for CD-ROM and CD image \
+access. Applications wishing to be oblivious of the OS- and device-dependent \
+properties of a CD-ROM or of the specific details of various CD-image formats \
+may benefit from using this library."
+HOMEPAGE="https://www.gnu.org/software/libcdio/"
+COPYRIGHT="2002-2014 Rocky Bernstein and Herbert Valerio Riedel"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="http://ftpmirror.gnu.org/libcdio/libcdio-$portVersion.tar.gz"
+CHECKSUM_SHA256="1b481b5da009bea31db875805665974e2fc568e2b2afa516f4036733657cf958"
+PATCHES="libcdio-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+
+PROVIDES="
+	libcdio$secondaryArchSuffix = $portVersion
+	lib:libcdio$secondaryArchSuffix = 18.0.0 compat >= 18
+	lib:libcdio++$secondaryArchSuffix = 1.0.0 compat >= 1
+	lib:libiso9660$secondaryArchSuffix = 11.0.0 compat >= 11
+	lib:libiso9660++$secondaryArchSuffix = 0.0.0 compat >= 0
+	lib:libudf$secondaryArchSuffix = 0.0.0 compat >= 0
+	"
+if [ -z "$secondaryArchSuffix" ]; then
+	PROVIDES="$PROVIDES
+		cmd:cdda_player
+		cmd:cd_drive
+		cmd:cd_info
+		cmd:cd_read
+		cmd:iso_info
+		cmd:iso_read
+		cmd:mmc_tool
```

Comment:
Do we add a ` = $portVersion` for each `cmd:*` entry in here?
(My understanding is that it is recommended to do so, but I guess we can keep this as is.)

Regarding the `libVersiion[Compat]` vars, maybe we should define `libcdioVersion`, `libcdioxxVersion`, `libiso9660Version`, `libiso9660xxVersion` and `libudfVersion` and use the eval "trick" to define the `lib*VersionCompat` the way we do it in e.g. `util_linux`.